services/web/server/src/simcore_service_webserver/exporter/file_downloader.py

```diff
@@ -38,19 +27,16 @@ async def append_file(self, link: str, download_path: Path):
 
     async def download_files(self, app: Application):
         """starts the download and waits for all files to finish"""
-
-        # run this async, parfive will support aiofiles in the future as stated above
-        wrapped_function = aiofiles_os.wrap(self.downloader.download)
         exporter_settings = get_settings(app)
-        results = await wrapped_function(
+        results = await self.downloader.run_download(
             timeouts={
                 "total": exporter_settings.downloader_max_timeout_seconds,
                 "sock_read": 90,  # default as in parfive code
             }
         )
-        log.debug("Download results %s", results)
 
-        if len(results) != self.total_files_added:
+        log.debug("Download results %s", results)
+        if len(results) != self.total_files_added or len(results.errors) > 0:
             raise ExporterException(
-                "Not all files were downloaded. Please check the logs above."
+                f"Not all files were downloaded. Please check the logs above. Errors: {results.errors}"
```

Comment:
MINOR: a priori, you do not know where this exception is going to be captured and handled, therefore the ``logs above`` is not very precise.

THOUGHT:
All messages, specially errors, should contain enough information to know what error, why/when it happened, how could it be solved (e.g. tip).

SUGGESTION:
Enhance the error message (and/or that of the ``log.debug`` of line 38) with information that can help debugging about  *what*/*why* is being downloaded (e.g. ``result.errors`` while downloading N files, ...), *when* (e.g. started) ,  *how* (e.g. download settings as timeout)...